.ci/run_metrics_ci.sh

```diff
@@ -87,9 +88,11 @@ pushd "$CURRENTDIR/../metrics"
 	bash storage/fio_job.sh -b 16k -o read -t "storage IO linear read bs 16k"
 	bash storage/fio_job.sh -b 16k -o write -t "storage IO linear write bs 16k"
 
+	# Pull request URL
+	PR="$GITHUB_URL/$LOCALCI_REPO_SLUG/pull/$LOCALCI_PR_NUMBER"
 
 	# Parse/Report results
-	emailreport
+	emailreport -c "Pull request: $PR"
```

Comment:
just a note - would be nice if we could get $LOCALCI_PR_NUMBER into the email subject line. Actually, probably more like:
`$LOCALCI_REPO_SLUG:$LOCALCI_PR_NUMBER` or similar. hmm, second thoughts - maybe 'github style' ?
`[$LOCAL_CI_REPO_SLUG] <report subject goes here> (#$LOCALCI_PR_NUMBER)`

thoughts?

Reply:
ok, I can add "-s" option in order to overwrite the subject in the email